src/platforms/common/configuration/options.mdx

```diff
@@ -352,7 +352,13 @@ Maximum number of characters a single value can have before it will be truncated
 
 <ConfigKey name="normalize-depth" supported={["javascript", "node"]}>
 
-Sentry SDKs normalize any contextual data to a given depth. Any keys containing data with a structure deeper than this will be trimmed and marked using its type instead (`[Object]` or `[Array]`), without walking the tree any further. By default, walking is performed `3` levels deep.
+Sentry SDKs normalize any contextual data to a given depth. Any data beyond this depth will be trimmed and marked using its type instead (`[Object]` or `[Array]`), without walking the tree any further. By default, walking is performed `3` levels deep.
```

Comment:
i know this isn't your edit, but the `3` is a bit of a cheat. I've done an edit that maintains the formatting, but another way to deal with it is: "By default, walking is performed three levels deep." Pick your poison! =)

```suggestion
Sentry SDKs normalize any contextual data to a given depth. Any data beyond this depth will be trimmed and marked using its type instead (`[Object]` or `[Array]`), without walking the tree any further. By default, the walking depth level is set to `3`.
```

Reply:
Fair. In our copious collective free time, we actually probably should go through this whole page and standardize how we specify defaults, if nothing else. They're pretty inconsistent.